ingestion_server/ingestion_server/state.py

```diff
@@ -27,6 +28,11 @@ class WorkerStatus(enum.Enum):
     ERROR = 2
 
 
+TaskData = namedtuple(
+    "TaskData", ["target_index", "task_id", "percent_successful", "percent_completed"]
+)
```

Comment:
I have to admit, I like [`typing.NamedTuple`](https://docs.python.org/3/library/typing.html#typing.NamedTuple) a bit more just because we can have types associated with it as well!
```suggestion
class TaskData(NamedTuple):
    target_index: int
    task_id: str
    percent_successful: float
    percent_completed: float
```

Reply:
Sounds good!